.github/workflows/test.yml

```diff
@@ -35,6 +35,12 @@ jobs:
       run: |
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
+        pip install --upgrade pytest-rerunfailures
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-scheming.git@release-3.0.0#egg=ckanext-scheming[requirements]'
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-harvest.git@v1.5.6#egg=ckanext-harvest[requirements]'
+        pip install -e 'git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-dcat.git@v1.5.1#egg=ckanext-dcat[requirements]'
+        pip install geomet>=0.2.0
+        pip install future>=0.18.2
```

Comment:
This two manual installs are not correct, they must be part of one of the extensions, so you need to install the extensions' `requirements.txt` instead.